packages/frontend-2/components/viewer/menu/Item.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <button
+    class="flex items-center justify-between hover:bg-highlight-1 text-foreground w-full h-full text-body-xs py-1 px-2 transition rounded-md"
```

Comment:
@andrewwallacespeckle Let's get rid of the transition on these menu items. I personally like when hovers like these are just instant. Else it can feel a bit lacky